modules/xfem/src/dirackernels/XFEMPressure.C

```diff
@@ -79,10 +44,10 @@ XFEMPressure::computeQpResidual()
   Real factor = _factor;
 
   if (_function)
-    factor *= _function->value(_t, _current_point);
+    factor *= _function->value(_t + _alpha * _dt, _current_point);
 
-  Point normal = _elem_qp_normal[_current_elem][_qp];
-  Real JxW = _elem_qp_JxW[_current_elem][_qp];
+  if (_postprocessor)
+    factor *= *_postprocessor;
 
-  return -factor * JxW * (normal(_component) * _test[_i][_qp]);
+  return factor * _interface_normal(_component) * _test[_i][_qp];
```

Comment:
@jiangwen84 In your original code, there was a minus sign. I removed the minus sign here and in the above definition of interface normal, so that it is consistent with tensor_mechanics, e.g. in Pressure.C.

Could you help me double check?

Reply:
Looks good to me.